Package.swift

```diff
@@ -0,0 +1,68 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let package = Package(
+    name: "FluentUI",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v13),
+        .macOS(.v10_14),
+    ],
+    products: [
+        .library(
```

Comment:
Does this mean the default export for macOS would be a static library too? Currently downstream we consume it via a framework packed inside the NuGet package, but I have some local code testing how it would look to use it as a static library for macOS instead.